src/components/Avatar/avatar.module.scss

```diff
@@ -70,5 +70,10 @@
   object-fit: cover;
 }
 
+.avatar-status-btn:hover {
```

Comment:
nit: use SASS syntax, consider other states like disabled, refer to button state matrix as a guide.

Reply:
We are not providing full fledged button for the status items. Hence not adding them. And, checked the way GitHub styles their avatar status items, it is the same.